.github/workflows/add-to-project.yml

```diff
@@ -14,5 +14,5 @@ jobs:
     steps:
       - uses: actions/add-to-project@v0.3.0
         with:
-          project-url: https://github.com/orgs/vmware-tanzu/projects/16
+          project-url: https://github.com/orgs/carvel-dev/projects/1
```

Comment:
Not much idea, but it seems this will add it in the `All` tab. 

Do we want to make it `https://github.com/orgs/carvel-dev/projects/6` which correspond to `kapp` tab? Although I do see earlier as well we were pointing to `All` tab.